src/modules/webhooks/WebhooksModule.ts

```diff
@@ -45,7 +46,9 @@ export default class WebhooksModule extends ConnectorRuntimeModule<WebhooksModul
         try {
             this.logger.debug(`Sending request to webhook '${url}' for trigger '${trigger}'.`);
 
-            const response = await this.axios.post(url, payload, { headers: webhook.target.headers });
+            const correlationId = correlator.getId();
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            const response = await this.axios.post(url, payload, { headers: { ...webhook.target.headers, "x-correlation-id": correlationId } });
```

Comment:
I think it would be safer like this. Also x-headers should be uppercase.

```suggestion
            let headers = webhook.target.headers;
            
            const correlationId = correlator.getId();
            // eslint-disable-next-line @typescript-eslint/naming-convention
            if (correlationId) headers = { ...headers, "X-CORRELATION-ID": correlationId } 
            
            const response = await this.axios.post(url, payload, { headers });
```

Reply:
Hi I took your fix but timo an I decided to keep the header lower case as headers are anyway case-insensitive and nodejs will anyway convert them to lower case.